common/hyperscaler/rules/definitions.go

```diff
@@ -0,0 +1,162 @@
+package rules
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/kyma-project/kyma-environment-broker/internal/broker"
+)
+
+const (
+	PR_ATTR_NAME = "PR"
+	HR_ATTR_NAME = "HR"
+	EU_ATTR_NAME = "EU"
+	S_ATTR_NAME  = "S"
```

Comment:
Could we use more verbose name like SHARED_ATTR_NAME?

Reply:
Yes, https://github.com/ralikio/kyma-environment-broker/commit/55aa6c282fa8b4cbb546d74cfd6576e5b59fbeeb